docs/docs/upgrade-gatsby-and-dependencies.md

```diff
@@ -0,0 +1,120 @@
+}
+```
+
+For major updates follow up with the corresponding guide from the [Release and Migrations](/docs/releases-and-migration/) reference guide overview.
+
+If you are updating Gatsby, you'll likely also need to update Gatsby related plugins, you can identify them by their names starting with `gatsby-`. This only applies to plugins managed in the gatsbyjs/gatsby repo; for community plugins check beforehand if there is a new version available for upgrading.
+
+## Updating all your dependencies at once
+
+After adding the corresponding annotations into your `package.json` file, you can run the update command:
+
+```shell
+npm update
+```
+
+This will upgrade all your packages to the latest [wanted version](https://docs.npmjs.com/cli/outdated), such as the latest patch, minor, or major update depending on your annotations in `package.json`.
+
+## Upgrade individual dependencies
+
+You can also update one package at the time with the `install` command in npm, alongside the version that you want to install:
+
+```shell
+npm install <package-name>@<version> --save
+```
+
+You can specify the version you want to install or upgrade to, in the following formats:
+
+- A specific version after the `@`
+- An annotated version with `*`,`^`,`~` to indicate that you want the latest major, minor or patch release respectively.
+- Use an `x` instead of a number to indicate that you want the latest major (`x`), minor (`<major>.x`) or patch release (`<major>.<minor>.x`). For example, to install the latest patch release for a given major and minor version: `npm install package-name@2.1.x --save`
+
+For major upgrades, remember to follow up with the corresponding guide from the [Release and Migrations](/docs/releases-and-migration/) reference guide overview.
+
+## Upgrade Interactively
+
+You can manually select which dependencies you want to update through the [npm-check](https://www.npmjs.com/package/npm-check) module. To do that, start by installing the module:
+
+```shell
+npm install npm-check --save-dev
+```
+
+Then add the corresponding script to your package.json file:
+
+```json:title=package.json
+{
+  "scripts": {
+    "upgrade-interactive": "npm-check --update"
+  }
+}
+```
+
+And finally run the recently added command:
+
+```shell
+npm run upgrade-interactive
+```
+
+## Troubleshooting
+
+Aside from some specific cases, such as [Gatsby's dropping of support for Node 6](/blog/2019-06-18-dropping-support-for-node-6/), upgrading for minor or patch releases should not require you to make changes to your code. It is recommended to run your suite of tests (in case you have one) after upgrading Gatsby or its dependencies.
+
+In case you get stuck in dependencies conflicts, you can use the [npm-force-resolutinos package](https://www.npmjs.com/package/npm-force-resolutions?activeTab=readme) on npm.
+
+## Related content
+
+Check out these related guides for major upgrades of Gatsby:
+
+[Migrating from v0 to v1](/docs/migrating-from-v0-to-v1/)
+
```

Comment:
```suggestion
```
Not 100% certain, but I believe removing this line break will facilitate grouping the migrating doc links into one list.